libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -92,7 +94,15 @@ const AP_Param::GroupInfo AP_Logger::var_info[] = {
     // @User: Standard
     // @Units: s
     AP_GROUPINFO("_FILE_TIMEOUT",  6, AP_Logger, _params.file_timeout,     HAL_LOGGING_FILE_TIMEOUT),
-    
+
+    // @Param: _MIN_MB_FREE
```

Comment:
This should have the `_FILE` bit in as it is only relevant for the file backend.